### ISSUE TITLE
Add support for .eq_cus files

### ISSUE THREAD

**Issue by zentol** [Synthlight/MHW-Editor#3]
Controls upgrade paths and costs.
```
{
    size = 41,
    offsetInitial = 10,
    entryCountOffset = 6,
    entries = new List<MhwStructData.Entry> {
        new MhwStructData.Entry("Item Category", 0, typeof(byte)),
        new MhwStructData.Entry("External Item Index", 1, typeof(ushort)),
        new MhwStructData.Entry("Unknown (int16) 1", 3, typeof(ushort)),
        new MhwStructData.Entry("Unknown (int32) 1", 5, typeof(int)),
        new MhwStructData.Entry("Story Unlock", 9, typeof(uint)),
        new MhwStructData.Entry("Item Rank", 13, typeof(uint)),
        new MhwStructData.Entry("Mat 1 Id", 17, typeof(ushort)),
        new MhwStructData.Entry("Mat 1 Count", 19, typeof(byte)),
        new MhwStructData.Entry("Mat 2 Id", 20, typeof(ushort)),
        new MhwStructData.Entry("Mat 2 Count", 22, typeof(byte)),
        new MhwStructData.Entry("Mat 3 Id", 23, typeof(ushort)),
        new MhwStructData.Entry("Mat 3 Count", 25, typeof(byte)),
        new MhwStructData.Entry("Mat 4 Id", 26, typeof(ushort)),
        new MhwStructData.Entry("Mat 4 Count", 28, typeof(byte)),
        new MhwStructData.Entry("Child index 1", 29, typeof(ushort)),
        new MhwStructData.Entry("Child index 2", 31, typeof(ushort)),
        new MhwStructData.Entry("Child index 3", 33, typeof(ushort)),
        new MhwStructData.Entry("Child Index 4", 35, typeof(ushort)),
        new MhwStructData.Entry("Unknown (uint16) 1", 37, typeof(ushort)),
        new MhwStructData.Entry("Unknown (uint16) 2", 39, typeof(ushort))
    }
});
```

The external item index refers for example to `l_sword.wp_dat`. I believe it is the _actual_ index in the file, oppose to the index in the [first byte](https://github.com/Synthlight/MHW-Editor/blob/master/MHW-Generator/Program.cs#L321).

The child index is the actual index of the child entry in the file; it would be immensely useful to have an additional index column; otherwise the moment you sort once the whole thing becomes difficult to use.
You can do some interesting things with this:

original:
![original](https://user-images.githubusercontent.com/5725237/73609646-12413980-45d0-11ea-88ad-c0d26e7d792b.png)

Modded:
![modded](https://user-images.githubusercontent.com/5725237/73609645-0eadb280-45d0-11ea-8d1c-ced84b5283e8.png)

**Comment by Synthlight** [comment]
As of right now there's no easy way to link dat files together. Maybe in the furute, but for now they stay as Ids.

Also, I cut out the last two unknowns. It didn't look like the data types were right. +38 looks like a single bytes that is just a 1/0 switch, but the rest just felt odd. Hard to tell either way so just gone for now till someone can identify the values.

**Comment by Synthlight** [comment]
Here's a few screenshots of the file assuming I got everything right.
Poke me if anything looks off.
![image](https://user-images.githubusercontent.com/1075166/73612216-ce423a80-45b7-11ea-962d-8b84aebf3387.png)
![image](https://user-images.githubusercontent.com/1075166/73612221-da2dfc80-45b7-11ea-9a87-4f4a10bb8614.png)

**Comment by Synthlight** [comment]
Done and released in v1.6.
